insalan/tournament/admin.py

```diff
@@ -63,11 +64,72 @@
     ]),
 ]
 
+class SeatCanvasWidget(forms.Widget):
+    """
+    Custom widget for the seat canvas
+    """
+    def render(self, name, value, attrs=None, renderer=None):
+        return '<canvas id="seatCanvas" width="500" height="500" ></canvas>'
```

Comment:
20x20 is too small, for comparison, this year's map is 37x21 (or 35x19 without borders)
![image](https://github.com/user-attachments/assets/d9b99de8-b95a-4521-b03c-da820c70b675)
I would recommand 50x50 so if someday the whole room is used for the tournament it will be possible. You can also reduce each pixel size.
Even better if you can, having the canvas auto-resizing (to have always 1 pixel border) will be the best.